sdk/core/src/error/http_error.rs

```diff
@@ -11,7 +11,7 @@ use serde::Deserialize;
 pub struct HttpError {
     status: StatusCode,
     details: ErrorDetails,
-    headers: Headers,
+    headers: impl Into<std::collections::HashMap<String, String>>,
```

Comment:
This is a breaking change. Please keep as-is.

Reply:
Thanks a lot, makes sense.
Changed as suggested